doc/aurbuild.md

```diff
@@ -0,0 +1,31 @@
+# aurbuild
+
+## Name
+
+aurbuild - Build packages to a local repository
+
+## Synopsis
+
+aurbuild [+-a ARG] [+-cd ARG] [+-p ARG] [+-r ARG} [--] ARGS...
+
+## Description
+
+Build a package from a queue file and add them to a local repository. A local repository is a repository on the local file system. It is configured using the file:// prefix for Server in pacman.conf.
```

Comment:
I may mention that the queue file includes directories which themselves contain a PKGBUILD.